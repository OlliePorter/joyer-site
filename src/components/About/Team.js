import React from 'react';
import Typography from '@material-ui/core/Typography';
import { makeStyles } from '@material-ui/core/styles';
import theme from '../../theme';

const useStyles = makeStyles({
  teamBackground: {
    backgroundColor: theme.palette.primary.altLight,
    padding: '3em',
  },
  title: {
    color: theme.palette.primary.altDark,
    fontWeight: 300,
    margin: '20px',
  },
});

export default function Team() {
  const classes = useStyles();

    return (
    <div className={classes.teamBackground}>
      <Typography component="h2" variant="h4" align="center" className={classes.title}>
        Our Team
      </Typography>
    </div>
    );
}
