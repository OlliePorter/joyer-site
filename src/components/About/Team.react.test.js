import React from 'react';
import Team from './Team';
import renderer from 'react-test-renderer';

it('Cover renders correctly', () => {
    const tree = renderer.create(
      <Team />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });