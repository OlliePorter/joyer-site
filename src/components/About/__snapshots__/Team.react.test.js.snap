// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cover renders correctly 1`] = `
<div
  className="makeStyles-teamBackground-1"
>
  <h2
    className="MuiTypography-root makeStyles-title-2 MuiTypography-h4 MuiTypography-alignCenter"
  >
    Our Team
  </h2>
  <div
    className="makeStyles-cardContainer-3"
  >
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-34 MuiPaper-rounded"
    >
      <div
        className="MuiCardMedia-root makeStyles-media-35"
        style={
          Object {
            "backgroundImage": "url(\\"/ollie.jpg\\")",
          }
        }
        title="Team profile icon"
      />
      <div
        className="MuiCardContent-root"
      >
        <h2
          className="MuiTypography-root makeStyles-nameStyle-37 MuiTypography-h5 MuiTypography-gutterBottom"
        >
          Ollie Porter
        </h2>
        <p
          className="MuiTypography-root makeStyles-titleStyle-38 MuiTypography-body1 MuiTypography-gutterBottom"
        >
          Co-Founder & CEO
        </p>
        <p
          className="MuiTypography-root makeStyles-bioStyle-39 MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Ollie is a certified Product Manager with three years experience within EdTech startups. He led the Onboarding process for Makers Academy, Europe’s leading web developer bootcamp, helped to implement Gett Inc.'s aggressive European growth strategy and graduated from NEF, the UK's most prestigious business course. He’s an avid meditator and mindfulness evangelist.
        </p>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-34 MuiPaper-rounded"
    >
      <div
        className="MuiCardMedia-root makeStyles-media-35"
        style={
          Object {
            "backgroundImage": "url(\\"/yas.jpg\\")",
          }
        }
        title="Team profile icon"
      />
      <div
        className="MuiCardContent-root"
      >
        <h2
          className="MuiTypography-root makeStyles-nameStyle-37 MuiTypography-h5 MuiTypography-gutterBottom"
        >
          Yas Green
        </h2>
        <p
          className="MuiTypography-root makeStyles-titleStyle-38 MuiTypography-body1 MuiTypography-gutterBottom"
        >
          Co-Founder & CTO
        </p>
        <p
          className="MuiTypography-root makeStyles-bioStyle-39 MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Yas is a full stack Software Developer at Hex Digital with previous experience writing clean and beautiful production code for The Economist. She’s a graduate of Makers Academy, with a degree in Forensic Science. She lives and breathes problem solving through code and forming creative solutions to some of the toughest problems. She’s excited about crafting the future of social through JOYER.
        </p>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
          />
        </svg>
      </div>
    </div>
    <div
      className="MuiPaper-root MuiPaper-elevation1 MuiCard-root makeStyles-card-34 MuiPaper-rounded"
    >
      <div
        className="MuiCardMedia-root makeStyles-media-35"
        style={
          Object {
            "backgroundImage": "url(\\"/ben.jpg\\")",
          }
        }
        title="Team profile icon"
      />
      <div
        className="MuiCardContent-root"
      >
        <h2
          className="MuiTypography-root makeStyles-nameStyle-37 MuiTypography-h5 MuiTypography-gutterBottom"
        >
          Ben Depla
        </h2>
        <p
          className="MuiTypography-root makeStyles-titleStyle-38 MuiTypography-body1 MuiTypography-gutterBottom"
        >
          Co-Founder & COO
        </p>
        <p
          className="MuiTypography-root makeStyles-bioStyle-39 MuiTypography-body1 MuiTypography-colorTextSecondary"
        >
          Ben is a Project Leader and Entrepreneur by trade, with a background in sales, finance, and real estate. He’s an efficiencies aficionado who's bringing fresh insight on how to best deliver on the JOYER vision. He studied languages at the University of Bristol and is a stickler for small details. He's looking for exponential growth through optimal processes and a gratified team. Bring on the joy!
        </p>
      </div>
      <div
        className="MuiCardActions-root MuiCardActions-spacing"
      >
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M22.46 6c-.77.35-1.6.58-2.46.69.88-.53 1.56-1.37 1.88-2.38-.83.5-1.75.85-2.72 1.05C18.37 4.5 17.26 4 16 4c-2.35 0-4.27 1.92-4.27 4.29 0 .34.04.67.11.98C8.28 9.09 5.11 7.38 3 4.79c-.37.63-.58 1.37-.58 2.15 0 1.49.75 2.81 1.91 3.56-.71 0-1.37-.2-1.95-.5v.03c0 2.08 1.48 3.82 3.44 4.21a4.22 4.22 0 0 1-1.93.07 4.28 4.28 0 0 0 4 2.98 8.521 8.521 0 0 1-5.33 1.84c-.34 0-.68-.02-1.02-.06C3.44 20.29 5.7 21 8.12 21 16 21 20.33 14.46 20.33 8.79c0-.19 0-.37-.01-.56.84-.6 1.56-1.36 2.14-2.23z"
          />
        </svg>
        <svg
          aria-hidden="true"
          className="MuiSvgIcon-root makeStyles-cardIcon-36"
          focusable="false"
          onClick={[Function]}
          role="presentation"
          viewBox="0 0 24 24"
        >
          <path
            d="M19 3a2 2 0 0 1 2 2v14a2 2 0 0 1-2 2H5a2 2 0 0 1-2-2V5a2 2 0 0 1 2-2h14m-.5 15.5v-5.3a3.26 3.26 0 0 0-3.26-3.26c-.85 0-1.84.52-2.32 1.3v-1.11h-2.79v8.37h2.79v-4.93c0-.77.62-1.4 1.39-1.4a1.4 1.4 0 0 1 1.4 1.4v4.93h2.79M6.88 8.56a1.68 1.68 0 0 0 1.68-1.68c0-.93-.75-1.69-1.68-1.69a1.69 1.69 0 0 0-1.69 1.69c0 .93.76 1.68 1.69 1.68m1.39 9.94v-8.37H5.5v8.37h2.77z"
          />
        </svg>
      </div>
    </div>
  </div>
</div>
`;
